{"name": "test_case8", "status": "failed", "statusDetails": {"message": "AssertionError: assert 1 == 3", "trace": "self = <testcases.test_allure_trend.TestCase03 object at 0x10595b820>\n\n    def test_case8(self):\n        print('case7...')\n>       assert 1 == 3\nE       assert 1 == 3\n\ntest_allure_trend.py:46: AssertionError"}, "start": 1737698191749, "stop": 1737698191749, "uuid": "53b6a88e-95e3-4134-9c67-ad30cd83e64f", "historyId": "fd077a06d932f5286c3da63b5f3964de", "testCaseId": "fd077a06d932f5286c3da63b5f3964de", "fullName": "testcases.test_allure_trend.TestCase03#test_case8", "labels": [{"name": "feature", "value": "测试类"}, {"name": "parentSuite", "value": "testcases"}, {"name": "suite", "value": "test_allure_trend"}, {"name": "subSuite", "value": "TestCase03"}, {"name": "host", "value": "bogon"}, {"name": "thread", "value": "89840-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "testcases.test_allure_trend"}]}